exporter/sumologicexporter/metadata.yaml

```diff
@@ -12,3 +12,8 @@ status:
 # TODO: Update the exporter to pass the tests
 tests:
   skip_lifecycle: true
+  goleak:
+    ignore:
+      top:
+        # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
+        - "go.opencensus.io/stats/view.(*worker).start"
```

Comment:
Fix is here

Reply:
@crobert-1 Looks good to me, but can you take a look at this exception before we merge this?

Reply:
Yep, looks good. The opencensus package starts a goroutine upon init that has no way to be shutdown, so just importing the package causes a leak.